### PR TITLE
Docs housekeeping for SEO and syntax, part 1

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR improves some of the technical aspects of our documentation. This is the first of a few, which will follow in the next week or so, mostly to continue adding meta descriptions to all existing docs that don't yet have them, or are problematic.

- Add meaningful meta descriptions (`description: "..."`) to more docs.
- Add GA tags to all docs that were missing them.
- Clean up old style frontmatter syntax.
  Before:
  ```
  <!--
  ---
  title: ...
  description: ...
  ---
  -->
  ```
  After:
  ```
  <!--
  title: ...
  description: ...
  -->
  ```

##### Component Name

area/docs

### CODE REVIEW

File: collectors/diskspace.plugin/README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <!--
----
 title: "diskspace.plugin"
+description: "Monitor the disk usage space of mounted disks in real-time with the Netdata Agent, plus preconfigured alarms for disks at risk of filling up."
```
Comment (aabatangle) [inline collectors/diskspace.plugin/README.md:3]: no need for a hyphen here in 'real time'

File: collectors/statsd.plugin/README.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <!--
----
 title: "statsd.plugin"
+description: "The Netdata Agent is a fully-featured statsd server that collects metrics from any custom application and visualizes them in real-time."
```
Comment (aabatangle) [inline collectors/statsd.plugin/README.md:3]: no need for a hyphen in 'fully featured' or 'real time', i think

**Review by aabatangle — APPROVED** [review]

**Comment by joelhans** [comment]
[Add analytics tags to everything that's missing them so far](https://trello.com/c/wMvd17OS/70-add-analytics-tags-to-everything-thats-missing-them-so-far)

**Review by aabatangle — DISMISSED** [review]

**Review by OdysLam — DISMISSED** [review]

**Comment by joelhans** [comment]
I had to rebase because there's been a lot of changes to docs with my other PRs merged today. No changes since the last approvals!

**Review by aabatangle — APPROVED** [review]

**Review by OdysLam — APPROVED** [review]

**Comment by joelhans** [comment]
[Do a full-on SEO audit and prioritize some docs-wide improvements](https://trello.com/c/1tFCsU8C/71-do-a-full-on-seo-audit-and-prioritize-some-docs-wide-improvements)

File: collectors/diskspace.plugin/README.md (hunk 1 above)
Comment (aabatangle) [inline collectors/diskspace.plugin/README.md:3]: disk usage space --> disk space usage?

File: collectors/statsd.plugin/README.md (hunk 2 above)
Comment (aabatangle) [inline collectors/statsd.plugin/README.md:3]: no hyphen needed for fully featured (or change to full-featured)

File: docs/configuration-guide.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+section. To disable specific modules of a plugin orchestrator, you need to edit one of the following:
+
+-   `python.d.conf` for [python](/collectors/python.d.plugin/README.md)
+-   `node.d.conf` for [nodejs](/collectors/node.d.plugin/README.md)
+-   `charts.d.conf` for [bash](/collectors/charts.d.plugin/README.md)
+
+#### Show charts with zero metrics
+
+By default, Netdata will enable monitoring metrics for disks, memory, and network only when they are not zero. If they
+are constantly zero they are ignored. Metrics that will start having values, after Netdata is started, will be detected
```
Comment (aabatangle) [inline docs/configuration-guide.md:90]: zero, they

File: docs/configuration-guide.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear
```
Comment (aabatangle) [inline docs/configuration-guide.md:91]: appear, though

File: docs/configuration-guide.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+documentation on how to define an alarm is in [health monitoring](/health/README.md). It is
+suggested to look at some of the stock alarm definitions, so you can ensure you understand how the various options work.
+
+#### Turn off all alarms and notifications
+
+Just set `enabled = no` in the [netdata.conf \[health\]](/daemon/config/README.md#health-section-options) section
+
+#### Modify or disable a specific alarm
+
+The `health.d` directory that contains the alarm triggers for [health monitoring](/health/README.md). It has
```
Comment (aabatangle) [inline docs/configuration-guide.md:111]: missing "is"?

File: docs/configuration-guide.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+one .conf file per collector. You can easily find the .conf file you will need to modify, by looking for the "source"
```
Comment (aabatangle) [inline docs/configuration-guide.md:112]: no need for a comma here; it's confusing.

File: docs/configuration-guide.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+line on the table that appears on the right side of an alarm on the Netdata gui.
```
Comment (aabatangle) [inline docs/configuration-guide.md:113]: GUI

File: docs/configuration-guide.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+to run `/etc/netdata/edit-config health.d/cpu.conf` and look for alarm at line 4 of the conf file. 
+
+As stated at the top of the .conf file, **you can disable an alarm notification by setting the 'to' line to: silent**.
+To modify how the alarm gets triggered, we suggest that you go through the guide on [health
+monitoring](/health/README.md#health-monitoring).
+
+#### Receive notifications using my preferred method
+
+You only need to configure `health_alarm_notify.conf`. To learn how to do it, read first [alarm
+notifications](/health/notifications/README.md#netdata-alarm-notifications) and then open the submenu `Supported
```
Comment (aabatangle) [inline docs/configuration-guide.md:127]: need comma before and

File: docs/configuration-guide.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+
+### Make security-related customizations
+
+#### Change the Netdata web server access lists
+
+You have several options under the [netdata.conf \[web\]](/web/server/README.md#access-lists) section. 
+
+#### Stop sending info to registry.my-netdata.io
+
+You will need to configure the `[registry]` section in `netdata.conf`. First read the [registry
```
Comment (aabatangle) [inline docs/configuration-guide.md:139]: First, read

File: docs/configuration-guide.md [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+documentation](/registry/). In it, are instructions on how to [run your own
```
Comment (aabatangle) [inline docs/configuration-guide.md:140]: no comma after In it

File: docs/configuration-guide.md [hunk 11]
```diff
@@ -0,0 +1,211 @@
...
+#### Move Netdata directories
+
+The various directory paths are in [netdata.conf \[global\]](/daemon/config/README.md#global-section-options).
+
+## How Netdata configuration works
+
+The configuration files are `name = value` dictionaries with `[sections]`. Write whatever you like there as long as it
+follows this simple format.
+
+Netdata loads this dictionary and then when the code needs a value from it, it just looks up the `name` in the
```
Comment (aabatangle) [inline docs/configuration-guide.md:174]: need comma before and (could also lose and and just have comma before 'then')

File: docs/configuration-guide.md [hunk 12]
```diff
@@ -0,0 +1,211 @@
...
+dictionary at the proper `section`. In all places, in the code, there are both the `names` and their `default values`,
```
Comment (aabatangle) [inline docs/configuration-guide.md:175]: no comma after places

File: web/api/queries/median/README.md [hunk 13]
```diff
@@ -1,8 +1,7 @@
 <!--
----
 title: "Median"
+description: "Use median in API queries and health entities to find the "middle" value from a sample, eliminating any unwanted spikes in the returned metrics."
```
Comment (aabatangle) [inline web/api/queries/median/README.md:3]: you've got double quotes inside double quotes; not sure if that's a problem per earlier issues

**Pull request merged**